### PR TITLE
[4] general file cleanup

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

File: administrator/components/com_privacy/src/View/Request/HtmlView.php [hunk 1]
```diff
@@ -82,7 +84,7 @@ public function display($tpl = null)
 		// Variables only required for the default layout
 		if ($this->getLayout() === 'default')
 		{
-			/** @var \ActionlogsModelActionlogs $logsModel */
+			/** @var ActionlogsModel $logsModel */
```
Comment (joomdonation) [inline administrator/components/com_privacy/src/View/Request/HtmlView.php]: I would change this to `/** @var \Joomla\Component\Actionlogs\Administrator\Model\ActionlogsModel $logsModel */` and remove the unnecessary use statement instead.

Comment (PhilETaylor) [reply administrator/components/com_privacy/src/View/Request/HtmlView.php]: Yes George likes it your way, but I subscribe to the group that Imports EVERY class that is needed, as having long namespaces throughout the code just looks messy to me :)

**Comment by PhilETaylor** [comment]
I have part reverted the `\JViewGenericdataexception` change here as that overlaps now with https://github.com/joomla/joomla-cms/pull/34005

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on bdbe701e535e2915e75d7476bdc42baf8034d62e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34019">issues.joomla.org/tracker/joomla-cms/34019</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on bdbe701e535e2915e75d7476bdc42baf8034d62e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34019">issues.joomla.org/tracker/joomla-cms/34019</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34019">issues.joomla.org/tracker/joomla-cms/34019</a>.</sub>

**Pull request merged**